### PR TITLE
Add additional event handlers for bounties.

### PR DESCRIPTION
**Pull request by jnaviask**

### CODE REVIEW

File: src/substrate/storageFetcher.ts [hunk 1]
```diff
@@ -359,36 +362,37 @@ export class StorageFetcher extends IStorageFetcher<ApiPromise> {
...
+            bountyIndex: +b.index,
+            value: b.bounty.value.toString(),
+            beneficiary: b.bounty.status.asPendingPayout.beneficiary.toString(),
+          } as ITreasuryBountyAwarded);
+          events.push({
+            kind: EventKind.TreasuryBountyClaimed,
+            bountyIndex: +b.index,
+            payout: b.bounty.value.toString(),
+            beneficiary: b.bounty.status.asPendingPayout.beneficiary.toString(),
+          } as ITreasuryBountyClaimed);
```
Comment (raykyri) [inline src/substrate/storageFetcher.ts:391]: the isPendingPayout state occurs before the bounty is claimed - does that make this incorrect?

File: src/substrate/filters/type_parser.ts [hunk 2]
```diff
@@ -198,7 +198,7 @@ export function ParseType(
           return EventKind.TreasuryBountyClaimed;
         case 'BountyCanceled':
           return EventKind.TreasuryBountyCanceled;
-        case 'BountyExtended':
+        case 'extendBountyExpiry':
```
Comment (raykyri) [inline src/substrate/filters/type_parser.ts:201]: Do we know why some methods are CamelCase event names, and others are actual method names?

File: package.json [hunk 3]
```diff
@@ -10,11 +10,11 @@
   "types": "dist/index.d.ts",
   "scripts": {
     "build": "tsc",
+    "prepare": "npx install-peers",
```
Comment (raykyri) [inline package.json:13]: Double checking: does npx install-peers work for you?

File: src/substrate/storageFetcher.ts (hunk 1 above)
Comment (jnaviask) [reply src/substrate/storageFetcher.ts:391]: I didn't change the logic at all, just moved the conditionals around. Perhaps it's always been incorrect and we should look into it.

File: src/substrate/filters/type_parser.ts (hunk 2 above)
Comment (jnaviask) [reply src/substrate/filters/type_parser.ts:201]: The methods triggered by events on the chain are CamelCase, but the ones triggered by extrinsics are the method names.

File: package.json (hunk 3 above)
Comment (jnaviask) [reply package.json:13]: Worked for me:
```
~/cw/chain-events$ yarn
yarn install v1.21.1
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info fsevents@2.1.3: The platform "linux" is incompatible with this module.
info "fsevents@2.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
[4/5] Linking dependencies...
warning " > @istanbuljs/nyc-config-typescript@0.1.3" has unmet peer dependency "source-map-support@*".
[5/5] Building fresh packages...
$ npx install-peers
- Installing 4 peerDependencies...
+ Successfully installed 4 peerDependencies via yarn.

yarn add v1.21.1
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info fsevents@2.1.3: The platform "linux" is incompatible with this module.
info "fsevents@2.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
[4/5] Linking dependencies...
[5/5] Building fresh packages...
success Saved 16 new dependencies.
info Direct dependencies
info All dependencies
├─ @polkadot/keyring@6.5.1
├─ @polkadot/networks@6.5.1
├─ @polkadot/wasm-crypto-asmjs@4.0.2
├─ @polkadot/wasm-crypto-wasm@4.0.2
├─ @polkadot/wasm-crypto@4.0.2
├─ @polkadot/x-fetch@6.5.1
├─ @polkadot/x-randomvalues@6.5.1
├─ @polkadot/x-textdecoder@6.5.1
├─ @polkadot/x-textencoder@6.5.1
├─ @polkadot/x-ws@6.5.1
├─ @types/websocket@1.0.2
├─ cuint@0.2.2
├─ ip-regex@4.3.0
├─ rxjs@6.6.7
├─ scryptsy@2.1.0
└─ xxhashjs@0.2.2
$ npx install-peers
Only run install-peer-deps once. Skipping.
Done in 15.09s.
Done in 22.48s.
```

File: src/substrate/storageFetcher.ts (hunk 1 above)
Comment (raykyri) [reply src/substrate/storageFetcher.ts]: Okay, fixed!

**Pull request merged**